From f0d9a2996837b492bf34fb2c464c85210a1cf20b Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 26 Dec 2018 22:00:51 +0100
Subject: [PATCH 1/1] efi_loader: use named constant for efi_dp_from_mem()

When calling efi_dp_from_mem() use a named constant for the memory type.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 cmd/bootefi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmd/bootefi.c b/cmd/bootefi.c
index f8f65c63f9d..d61609e84c0 100644
--- a/cmd/bootefi.c
+++ b/cmd/bootefi.c
@@ -350,7 +350,7 @@ static efi_status_t do_bootefi_exec(void *efi,
 	if (!device_path && !image_path) {
 		printf("WARNING: using memory device/image path, this may confuse some payloads!\n");
 		/* actual addresses filled in after efi_load_pe() */
-		memdp = efi_dp_from_mem(0, 0, 0);
+		memdp = efi_dp_from_mem(EFI_RESERVED_MEMORY_TYPE, 0, 0);
 		device_path = image_path = memdp;
 		/*
 		 * Grub expects that the device path of the loaded image is
-- 
2.19.2

